Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -526,6 +526,11 @@ static ASCIILiteral serviceName(const ProcessLauncher::LaunchOptions& launchOpti
 
 void ProcessLauncher::terminateXPCConnection()
 {
+#if USE(EXTENSIONKIT)
+    if (m_process)
```

Comment:
Seems the rest of the code using the xpc_connection should likely be called in an `#else` case since I don't believe it does anything when using EXTENSIONKIT.

Reply:
Ah, these a are good catches, I will address in the next patch.

Thanks for reviewing!